### PR TITLE
Improve compatibility with ITK 4.13

### PR DESCRIPTION
**Pull request by dzenanz**
This allows building in 4.13.0, both inside and outside of ITK.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -47,6 +47,7 @@ include_directories(${CMAKE_BINARY_DIR})
 if(NOT ITK_SOURCE_DIR)
   include(ITKModuleExternal)
 else()
+  set(ITK_DIR ${CMAKE_BINARY_DIR})
```
Comment (dzenanz) [inline CMakeLists.txt]: @thewtex Why is this required? Without this some random ITK installation is usually found.

Comment (thewtex) [reply CMakeLists.txt]: If built from `ITK/Modules/Remote`, we want `find_package(ITK` in `PerformanceBenchmarking/examples` to find the ITK we are building.

Comment (dzenanz) [reply CMakeLists.txt]: Should that be in the [template](https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/CMakeLists.txt) then?

Comment (thewtex) [reply CMakeLists.txt]: Yes.
